packages/sit-onyx/src/i18n/locales/ko-KR.json

```diff
@@ -0,0 +1,21 @@
+{
+  "optional": "(optional)",
```

Comment:
Hey Kai, thanks for the contribution!
Is there also a korean translation for "optional" to indicate that a form component is not required? :)
![image](https://github.com/SchwarzIT/onyx/assets/67898185/c181725e-c057-4ac4-8bd5-06a0e21ec672)

Reply:
@larsrickert Hello! Lars!
Oh you are right, I missed the part.